region/wreckedship/main/Wrecked Ship East Super Room.json

```diff
@@ -707,6 +708,41 @@
       "farmCycleDrops": [{"enemy": "Covern", "count": 1}],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Pause Abuse Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"or": [
+          {"and": [
+            "h_CrystalFlashForReserveEnergy",
+            "canTrickyDodgeEnemies"
```

Comment:
I don't see how you could avoid killing the bulls

Reply:
I had at one point in testing managed to get a CF that killed only one Bull. But since pause abuse is easier anyway...